phoenix-core/src/main/java/org/apache/phoenix/compile/UpsertCompiler.java

```diff
@@ -175,8 +175,10 @@ private static void setValues(byte[][] values, int[] pkSlotIndex, int[] columnIn
         mutation.put(ptr, new RowMutationState(columnValues, columnValueSize, statement.getConnection().getStatementExecutionCounter(), rowTsColInfo, onDupKeyBytes));
     }
     
-    public static MutationState upsertSelect(StatementContext childContext, TableRef tableRef, RowProjector projector,
-            ResultIterator iterator, int[] columnIndexes, int[] pkSlotIndexes, boolean useServerTimestamp, boolean prefixSysColValues) throws SQLException {
+    public static MutationState upsertSelect(StatementContext childContext, TableRef tableRef,
```

Comment:
This class was just formating only correct?

Reply:
Mostly. Apart from formatting, removed some dead variable initialization before a return (MutationState `state` variable) as well.